### ISSUE TITLE
[Feature request] Support learning rate warm-up steps and schedulers

### ISSUE THREAD

**Issue by wangz10** [awslabs/graphstorm#306]
Fine-tuning LM and co-training involving LM often requires LR schedulers. It would be nice to support various LR scheduler to reproduce SOTA results.
